src/coreclr/vm/common.h

```diff
@@ -181,7 +181,11 @@ typedef PTR_Object OBJECTREF;
 typedef DPTR(OBJECTREF) PTR_OBJECTREF;
 typedef DPTR(PTR_OBJECTREF) PTR_PTR_OBJECTREF;
 
-EXTERN_C Thread* STDCALL GetThread();
+Thread* GetThread();
+Thread* GetThreadNULLOk();
+
+EXTERN_C Thread* STDCALL JitGetThreadHelper();
```

Comment:
This does not have much to do with Jit. Would it be better to call it just `GetThreadHelper`?

Other potential name ideas: `_GetThread`, `GetThreadStatic`, ... .

Reply:
It is basically just `GetThreadNULLOk` that is not `inline` to be used from asm and generated code. `inline` functions are not always there and linker gets unhappy.
I wanted the helper to have a name that makes it clear it is not to be used in regular code. It was `PInvokeGetThreadHelper` at some point, since most uses were in stubs, but that were not all uses, so I changed to JIt...

`GetThreadHelper` sounds good enough.